source/operations/concepts.rst

```diff
@@ -186,7 +186,12 @@ Some common reasons for bit rot include:
 - accidental overwrites
 
 MinIO uses a hashing algorithm to confirm the integrity of an object.
-If an object becomes corrupted by bit rot, MinIO can automatically heals the object depending on the system topology and availability.
+This algorithm automatically applies at the time of any ``GET`` request that calls an object.
+If an object becomes corrupted by bit rot, MinIO can automatically :ref:`heal <minio-concepts-healing>` the object depending on the availability of parity shards for the object.
+
+While bit rot healing can be done during the routine checks of the :ref:`MinIO Scanner <minio-concepts-scanner>`, this function is off by default.
```

Comment:
SO I think we need to make this clearer.

We do a bitrot/health check on `GET` or `HEAD` (I think).

cc/ @harshavardhana @klauspost to confirm.

This clarity is important so we don't have users walking away thinking we never do bitrot checks - we just do not do them as part of the _scanner_, which itself _does_ do high level health checks.

Reply:
Confirmed with Harsha and updated the wording here.